crates/vault-registry/src/lib.rs

```diff
@@ -1544,6 +1561,20 @@ impl<T: Config> Pallet<T> {
         collateral.try_into().map_err(|_| Error::<T>::TryIntoIntError.into())
     }
 
+    pub fn get_max_nomination_ratio() -> Result<UnsignedFixedPoint<T>, DispatchError> {
```

Comment:
See comment on usage, I don't think we need this here.

Reply:
This needs to be called when checking if withdrawals exceed the Max Nomination Ratio